src/routes/forgot-password/components/ForgotPassword.js

```diff
@@ -0,0 +1,103 @@
+const React = require('react');
+const T = require('prop-types');
+const StrangeForms = require('strange-forms');
+const IsEmail = require('utils/is-email');
+
+module.exports = class ForgotPassword extends StrangeForms(React.Component) {
+
+    static propTypes = {
+        requestReset: T.func.isRequired,
+        errorMessage: T.string
+    };
+
+    constructor(props) {
+
+        super(props);
+
+        this.state = {
+            email: '',
+            isBlurred: {
+                email: false
+            }
+        };
+
+        this.strangeForm({
+            fields: ['email'],
+            get: (someProps, field) => this.state[field],
+            act: this.setFormField.bind(this),
+            getFormValue: this.getFormValue.bind(this)
```

Comment:
This could be defined inline too!  Actually you probably don't need to configure it at all—I think this implementation is strange-forms's default.

Reply:
yep, great point! i think this was leftover from when i initially configured this to accept stuff from props and state on a different form (that I lazily copied and pasted!). So I'll update all the forms to be defined inline, yes?

Reply:
Sounds good!